internal/provider/models/team.go

```diff
@@ -64,3 +80,55 @@ func (data *Team) ReadFromResponse(ctx context.Context, team *iam.Team) diag.Dia
 
 	return nil
 }
+
+func (data *TeamResource) ReadFromResponse(ctx context.Context, team *iam.Team, memberIds *[]string) diag.Diagnostics {
+	var diags diag.Diagnostics
+	data.Id = types.StringValue(team.Id)
+	data.Name = types.StringValue(team.Name)
+	if team.Description != nil && *team.Description != "" {
+		data.Description = types.StringValue(*team.Description)
+	} else {
+		data.Description = types.StringNull()
+	}
```

Comment:
wrote it this way because i kept hitting `Provider produced inconsistent result after apply`